go/abi/DepositLog.go

```diff
@@ -0,0 +1,2406 @@
+// Code generated - DO NOT EDIT.
```

Comment:
Can we keep all generated files inside `gen` directory (`go/gen`)? This is the pattern we use everywhere no matter what's generated (commands, bindings, promises).

Reply:
But in `keep-ecdsa` we have the same structure: https://github.com/keep-network/keep-ecdsa/tree/master/pkg/chain/gen

Reply:
Right, and what I am talking about it this `/gen/` part :)

Reply:
Ah, got it now! I'll change it.